internal/cmd/flink/command_statement_describe.go

```diff
@@ -0,0 +1,52 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newStatementGetDetailsCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "describe <name>",
+		Short:             "Get details of a Flink SQL statement.",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validStatementArgs),
+		RunE:              c.statementDescribe,
+	}
+
+	c.addComputePoolFlag(cmd)
```

Comment:
One more thing: maybe I'm not overseeing it, but I don't think the compute pool flag is used anywhere